### PR TITLE
Floor switching

### PR DESCRIPTION
**Pull request by Tristanas**
Implemented floor switching. Added 1 more fake floor to test if this functions properly. Improved the display of identified people per camera. Created a method for fake data generation whenever you click on a camera.

### CODE REVIEW

File: SafePlace/Service/CameraService.cs [hunk 1]
```diff
@@ -18,7 +18,10 @@ public Camera CreateCamera()
                 Guid = Guid.NewGuid(),
                 Transform = new System.Windows.Media.TransformGroup(),
                 //Test data for implementing ItemSource, which takes items from ObservableCollection<Person>
-                IdentifiedPeople = new List<Person>() { new Person() { Name = "John", LastName = "Johnson"} }
+                IdentifiedPeople = new List<Person>() {
```
Comment (TMikna) [inline SafePlace/Service/CameraService.cs]: Now looks much better

File: SafePlace/Service/CameraService.cs [hunk 2]
```diff
@@ -18,7 +18,10 @@ public Camera CreateCamera()
...
+                    new Person() { Name = "John", LastName = "Johnson" },
+                    new Person() { Name = "John", LastName = "Seenhim" }
```
Comment (TMikna) [inline SafePlace/Service/CameraService.cs]: Now looks much better

File: SafePlace/Views/HomePageView/HomePagePresenter.cs [hunk 3]
```diff
@@ -47,9 +47,7 @@ private void BuildViewModel()
             {
                 _viewModel.FloorList.Add(floor.FloorName);
             }
-            _viewModel.SpottedPeople = new ObservableCollection<Person>();
             //Initializing Cameras observable collection, which will be used to store names, connecting images to cameras
```
Comment (TMikna) [inline SafePlace/Views/HomePageView/HomePagePresenter.cs]: This comment looks no more relevant here

File: SafePlace/Views/HomePageView/HomePagePresenter.cs [hunk 4]
```diff
@@ -110,7 +111,16 @@ public Image ImageFromCoords(int x, int y)
...
-        
+        public void LoadFloor(Floor NewFloor)
+        {
+            _viewModel.CurrentFloorImage = NewFloor.FloorMap;
+            ReloadObservableCollection(_viewModel.Cameras, NewFloor.Cameras);
+        }
 
+        public void ReloadObservableCollection<T>(ObservableCollection<T> oCollection, IList<T> list)
+        {
+            oCollection.Clear();
```
Comment (TMikna) [inline SafePlace/Views/HomePageView/HomePagePresenter.cs]: oCollection - not so understandable

**Comment by TMikna** [comment]
Left some comments, check them. Overall looks good

File: SafePlace/Views/HomePageView/HomePagePresenter.cs (hunk 4 above)
Comment (Tristanas) [reply SafePlace/Views/HomePageView/HomePagePresenter.cs]: observableColl should be good enough

**Pull request merged**